src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -5550,6 +5550,37 @@ @inherits BaseComponent<string?>
             ]);
     }
 
+    [IntegrationTestFact, WorkItem("https://github.com/dotnet/razor/issues/10863")]
+    public void PageDirective_NoForwardSlash()
+    {
+        // Act
+        var generated = CompileToCSharp("""
+            @page "MyPage"
```

Comment:
I hate to ask, but what about `@page /* Comment */ "/MyPage"`, or just `@page /* Comment */`?

Reply:
Oh huh, thats a good point. Let me add tests.

Reply:
Added tests, we still function correctly as in the no route case we're malformed, so don't go down this path. In the comment case we remove the comment, so we only have a single token.